blog/2024-11-25-camera-deprecations.md

```diff
@@ -0,0 +1,16 @@
+---
+author: Robert Resch
+authorURL: https://github.com/edenhaus
+authorImageURL: https://avatars.githubusercontent.com/u/26537646?s=96&v=4
+title: "Camera deprecations"
```

Comment:
Camera deprecations sounds very generic and could be seen as "cameras will be deprecated". Can we think of a slightly better name?

Reply:
What do you think about:
- Camera methods deprecation
- Camera refactoring

Reply:
I think all 3 code rabbit ones would be good